### DISCUSSION TITLE
Why Aim part of Performance Points so strongly depends on Accuracy?

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#22288] [Q&A] [answered]
I found something illogical while playing with Relax mod. 

FC Relax play on map 5.49* (7.26* without relax) 95.28% accuracy is only **7 of 173** pp. (**96%** of PP were absorbed for "bad" accuracy). 
BM link: https://osu.ppy.sh/beatmapsets/1630848#osu/3400264
Screen:
![osu_2023-01-19_07-11-45](https://user-images.githubusercontent.com/55319814/213364226-f7e9a30a-a113-4097-b4d0-98bc6a7fe874.jpg)

Let's see on another case with similar starrate.
Hitorigoto... Easy TV-size farm map. The play of the same level.
5.86* with Relax (overrated as for Relax, I think) 7.20* originally. 99.37% accuracy **177/188**pp (**6%** were absorbed for accuracy)
BM link: https://osu.ppy.sh/beatmapsets/596704#osu/1262832
Screen: 
![osu_2023-01-19_07-24-51](https://user-images.githubusercontent.com/55319814/213365301-8a9e0de1-cd9d-41ce-8580-335d0f766f21.jpg)

Both plays were on approximately the same level. Both FC. For long and more difficult Melody Flag 7pp and 177pp for Hitorigoto.

I really confused by it. **How exactly is such a result calculated and why can the results be so different from each other?**

**Comment by bdach** [comment]
cc/ @ppy/osu-pp-committee

**Comment by stanriders** [comment, marked as answer]
With relax every ok/meh counts partly as a miss which also scales with OD (higher OD - smaller penalty), hence why it reduces pp so harshly.

**Reply by okolenmi** [reply]
Thank you for answer. Now I want to say why I do not respect this approach and why it is absurd. I don't expect to be heard. It's just that my entry should exist in case this topic is of interest to **people who are really concerned about game balance**. Anyway, my answer is short:

It's clearly penalty for map duration. Long map = less pp even with the same accuracy relatively to short map (because you have more 100's). Also it's strange to think player can't hit object after late aim (I can do it almost always, not to mention more skilled players)... Also player with relax can't hit object early - this restriction looks like OD*2 as half of hit line unavailable.

**Reply by stanriders** [reply]
Balancing relax isn't really a priority in any way, sorry. 

Accuracy doesn't play any role here as it's supposed to represent tapping accuracy, not aiming accuracy. The only concept of gauging how good your score is is "you've hit an object or you missed".

The concept behind the penalty is quite simple - if you hit early (which can happen if you take your cursor away from the object before the great hit window) or late, it's a major misaim and should be punished. Relax is all about aim so perfect aim coordination is rewarded and "mashing" is not.

**Reply by bdach** [reply]
In addition: @okolenmi please cut the incendiary language ("...absurd", "...of interest to people who are really concerned about game balance"). We're not on twitter here. I know that as soon as pp points are involved debates tend to get rather heated, but you will not get treated seriously here if you say that sort of stuff.
